packages/fcl/src/current-user/index.js

```diff
@@ -100,8 +101,8 @@ async function authenticate(opts = {}) {
     spawnCurrentUser()
     const user = await snapshot()
     if (user.loggedIn && notExpired(user)) return resolve(user)
-    const serviceStrategy = opts.serviceStrategy || frame
-
+    const serviceStrategy =
+      (await config.first(["serviceStrategy"], "pop")) === "frame" ? frame : pop
```

Comment:
Can we remove this for now and just default it to `pop`

Reply:
updated to default to `pop()`